README.md

```diff
@@ -9,8 +9,10 @@ some new part of the compiler that they haven't worked on before.
 [You can read the latest version of the guide here.](https://rustc-dev-guide.rust-lang.org/)
 
 You may also find the rustdocs [for the compiler itself][rustdocs] useful.
+Note that these are not intended as a guide; it's recommended that you search
+for the page you're looking for instead of reading them top to bottom.
```

Comment:
Maybe say this or this?

```suggestion
for the item you're looking for instead of reading them top to bottom.
```

```suggestion
for the docs you're looking for instead of reading them top to bottom.
```

Saying 'the page' sounds like we're suggesting you search the dev-guide, which may be a good suggestion anyway but I don't think it's what you meant.

Reply:
I went with `docs`, thanks!